backend/src/routes/notes.ts

```diff
@@ -0,0 +1,23 @@
+import express, { NextFunction, Request, Response } from "express";
+import { Note } from "../models/notes";
+
+const router = express.Router();
+
+router.get("/notes/:id", async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    console.log("Getting...");
+    const id = req.params.id;
+    const note = await Note.findById(id);
+    if (note == null) {
+      throw new Error();
+    }
+    return res.status(200).json(note.answers);
+  } catch (e) {
+    next(e);
```

Comment:
Just letting you know that next(e) isn't actually doing anything yet cause we don't have a error handler function and we've just been coding the handlers in the catch block itself. Although we should prolly at some point create an error handler function that is used for all routes.